TestApp/Enums.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TestApp
+{
+    public enum FoodType
```

Comment:
bad file naming
fix
way to resolve - two enums - two files - can be in some separatew folder